app/react/settings/people/PeopleNavigationContainer.jsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+const PeopleNavigationContainer = () => (
+    <div className="mode-context init">
+        <header className="mode-context-header">
+            <ol className="breadcrumbs">
+                <li>
+                    <a href="/" ng-controller="navigation as nav">
+                        nav.site.name
+                    </a>
+                </li>
+                <li>
+                    <a href="/settings" translate>
```

Comment:
Are you trying to use translate the same way we use it in Angular here?

Reply:
nope I just did some copy-pasting html and this stayed. Removing